charts/alertmanager-snmp-notifier/README.md

```diff
@@ -0,0 +1,71 @@
+# Prometheus SNMP Notifier
+
+An Alertmanager webhook that relays alerts as SNMP traps.
+
+This chart creates a [SNMP Notifier](https://github.com/maxwo/snmp_notifier) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.8+ with Beta APIs enabled
```

Comment:
This seems kinda old. 
Was this testesd onm this old version?
I would rather start with some k8s version which is not already eol...

Reply:
Indeed. I reused another chart as a base in that same repository, thinking I would comply with some rules of this helm repository.
I changed it with the version I could properly test, but it should be compatible with much older versions.